draft-ietf-quic-qlog-quic-events.md

```diff
@@ -106,20 +106,16 @@ trace of the connection with ODCID abcd1234).
 
 ## Raw packet and frame information
 
-Note:
-
-: QUIC packets always include an AEAD authentication tag at the end.
-As this tag is always the same size for a given connection (it depends on the
-used TLS cipher suite), this document does not define a separate
-"RawInfo:aead_tag_length" field here. Instead, this field is reflected in
-"transport:parameters_set" and can be logged only once.
+QUIC packets always include an AEAD authentication tag at the end.
+In general, the length of the AEAD tag depends on the TLS cipher
+suite, although all cipher suites used for QUIC use a 16 byte tag.
```

Comment:
I wonder if we should be explicit in saying "used for QUIC v1" or "used in RFC9001" for future-proofness?